src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -1111,18 +1111,12 @@ void SpatialPooler::inhibitColumns_(vector<Real>& overlaps,
     density = min(density, (Real) 0.5);
   }
 
```

Comment:
It's not used anymore, but I was thinking of removing it in a separate PR. Should I remove it in this one?